Mage/src/main/java/mage/abilities/dynamicvalue/MultipliedValue.java

```diff
@@ -33,23 +39,49 @@ public MultipliedValue copy() {
         return new MultipliedValue(this);
     }
 
+//    @Override
+//    public String toString() {
+//        if (value.toString().equals("1")) {
+//            return Integer.toString(multiplier);
+//        }
+//        StringBuilder sb = new StringBuilder();
+//        if (multiplier == 2) {
```

Comment:
Do not keep commented code or add explanation about it in the first line like "old version" or "TODO: need check all used cards"